File: Java Barcode Scanner ME/app/src/main/java/de/dimedis/mobileentry/backend/response/UserPrefs.java
package de.dimedis.mobileentry.backend.response;

import com.google.gson.annotations.SerializedName;

public class UserPrefs {
    public static final String KEYS = "keys";

    public UserPrefs() {
        mKeys = new Keys();
    }

    @SerializedName(KEYS)
    private Keys mKeys;

    public Keys getKeys() {
        return mKeys;
    }
}

File: Kotlin Migration/NTMobileEntry/app/src/main/java/de/dimedis/mobileentry/backend/response/UserPrefs.kt
package de.dimedis.mobileentry.backend.response

import com.google.gson.annotations.SerializedName

class UserPrefs {

    @SerializedName(KEYS)
    private val mKeys: Keys = Keys()

    val keys: Keys
        get() = mKeys

    companion object {
        const val KEYS = "keys"
    }
}